cardform/src/main/java/com/mercadolibre/android/cardform/network/interceptor/AuthorizationInterceptor.kt

```diff
@@ -0,0 +1,21 @@
+package com.mercadolibre.android.cardform.network.interceptor
+
+import okhttp3.Interceptor
+import okhttp3.Response
+import java.io.IOException
+
+private const val AUTHORIZATION_HEADER = "Authorization"
+
+internal class AuthorizationInterceptor(private val privateKey: String?) : Interceptor {
+
+    @Throws(IOException::class)
+    override fun intercept(chain: Interceptor.Chain): Response {
+        return chain.proceed(chain.request().let { request ->
+            privateKey?.let {
```

Comment:
El check lo podemos hacer antes de agregar el interceptor?

Ej:

`accessToken?.also { client.addInterceptor(AuthorizationInterceptor(it) }`

Reply:
listo